src/components/CodeOverview/index.tsx

```diff
@@ -21,6 +24,8 @@ import {
   generateLineShapes,
 } from '../CodeLineShapes/utils';
 
+export type GetCodeLineAnchor = (line: number) => string;
```

Comment:
If this is related to `getCodeLineAnchor`, shouldn't this type be defined in `CodeView/utils.tsx`?

Reply:
Also, what would be the difference between this type and `typeof getCodeLineAnchor`? Or could we define this type as follows:

```
type GetCodeLineAnchor = typeof getCodeLineAnchor;
```

Reply:
> ```
> type GetCodeLineAnchor = typeof getCodeLineAnchor;
> ```

That works well. Thanks.